Extracting reference data (Isolation forest).ipynb

```diff
@@ -0,0 +1,164 @@
+{
```

Comment:
<p>Nice, this will mesh well with how plots are saved in <a href="https://github.com/BU-Spark/pitne-season-watch/pull/15/" target="_blank">#15</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/pitne-season-watch/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>